File: src/main/java/com/kelsos/mbrc/commands/NotifyPluginOutOfDateCommand.java
package com.kelsos.mbrc.commands;

import com.google.inject.Inject;
import com.kelsos.mbrc.interfaces.ICommand;
import com.kelsos.mbrc.interfaces.IEvent;
import com.kelsos.mbrc.messaging.NotificationService;

public class NotifyPluginOutOfDateCommand implements ICommand {
    @Inject NotificationService notificationService;

    @Override public void execute(IEvent e) {
        notificationService.updateAvailableNotificationBuilder();
    }
}

File: app/src/main/kotlin/com/kelsos/mbrc/commands/NotifyPluginOutOfDateCommand.kt
package com.kelsos.mbrc.commands

import com.kelsos.mbrc.interfaces.ICommand
import com.kelsos.mbrc.interfaces.IEvent
import com.kelsos.mbrc.messaging.NotificationService
import javax.inject.Inject

class NotifyPluginOutOfDateCommand
@Inject constructor(private val notificationService: NotificationService) : ICommand {

  override fun execute(e: IEvent) {
    //notificationService.updateAvailableNotificationBuilder();
  }
}
